shims/spark311/src/main/scala/com/nvidia/spark/rapids/shims/spark311/ParquetCachedBatchSerializer.scala

```diff
@@ -217,9 +219,15 @@ private object ParquetCachedBatch {
   def apply(parquetBuff: ParquetBufferConsumer): ParquetCachedBatch = {
     new ParquetCachedBatch(parquetBuff.numRows, parquetBuff.getBuffer)
   }
+
+  def apply(numRows: Int, buff: Array[Byte]): ParquetCachedBatch = {
```

Comment:
If `ParquetCachedBatch` is a case class why do we need this? It should automatically be generated for us.